### DISCUSSION TITLE
web_demo.py聊天存在js报错导致无法正常聊天

### DISCUSSION THREAD

**Discussion by rongdi** [zai-org/ChatGLM3#507] [Bad Case]
![image](https://github.com/THUDM/ChatGLM3/assets/5162800/3d85aea7-7faf-4d9b-bde0-99ad53c2a7b1)

聊天在页面没有显示，但是实际上ws中正常返回了回答
![image](https://github.com/THUDM/ChatGLM3/assets/5162800/b911df41-f027-4ec8-a798-b198e5e44628)

存在js报错，如下
![image](https://github.com/THUDM/ChatGLM3/assets/5162800/ff10e5fd-65e2-4861-bb86-390f227904b9)

**Comment by zRzRzRzRzRzRzR** [comment]
版本不对，3.39的gradio
